deployment/network-operator/README.md

```diff
@@ -334,6 +334,7 @@ Production cluster environment can deny direct access to the Internet and instea
 | `ofedDriver.image` | string | `mofed` | Mellanox OFED driver image name |
 | `ofedDriver.version` | string | `5.5-1.0.3.2` | Mellanox OFED driver version |
 | `ofedDriver.imagePullSecrets` | list | `[]` | An optional list of references to secrets to use for pulling any of the Mellanox OFED driver image |
+| `ofedDriver.env` | list | `[]` | An optional list of environment variables passed the Mellanox OFED driver image                    |
```

Comment:
`An optional list of environment variables passed to the Mellanox OFED driver container` instead of
`An optional list of environment variables passed the Mellanox OFED driver image`

Reply:
please add a link to : https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core

or similar so the format is clear